modules/katamari/src/test/ts/atomic/api/arr/ArrArrayLikeTest.ts

```diff
@@ -1,5 +1,7 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
-import * as Arr from 'ephox/katamari/api/Arr';
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+import { Arr } from 'ephox/katamari/api/Main';
```

Comment:
```suggestion
import * as Arr from 'ephox/katamari/api/Arr';
```
I could have sworn we had an ESLint rule that detected this... it may need some tweaking.